internal/detectors/dependencies/dependencies.go

```diff
@@ -40,6 +40,54 @@ func (detector *detector) AcceptDir(dir *file.Path) (bool, error) {
 	return true, nil
 }
 
+func DetectorsForLanguage(language string) []string {
+	switch language {
+	case "C#":
```

Comment:
Wondering if we prefer to add language information to the dependencies directly, and avoid this case statement and maintenance thereof. Any thoughts?

Reply:
^ After discussion, we will add language information to the dependencies, and ensure that this matches the gocloc result (i.e. update `"Javascript"` to `"JavaScript"`)